src/pages/auth/RegistrationPage.tsx

```diff
@@ -18,124 +18,132 @@ import { getRel8UserRegistrationData} from "../../utils/extra_functions";
 import { useMutation } from "react-query";
 import { createMember } from "../../api/auth/auth-api";
 import Button from "../../components/button/Button";
-
+import { toast } from "react-toastify";
 
 export type RegistrationFormFields = {
-  fullname:string;
+  fullname: string;
   username: string;
-  rel8Email:string;
+  rel8Email: string;
   password: string;
-  phone_number:string;
-  department:string;
-  graduation_year:string;
-  chapter:string;
-  MEMBERSHIP_NO:string;
+  phone_number: string;
+  department: string;
+  graduation_year: string;
+  chapter: string;
+  MEMBERSHIP_NO: string;
 };
 
-
 const RegistrationPage = () => {
-  const data:any = getRel8UserRegistrationData()
-
+  const data: any = getRel8UserRegistrationData();
   const { notifyUser } = Toast();
   const navigate = useNavigate();
-
   const { mutate, isLoading } = useMutation(createMember, {
     onSuccess: (data) => {
-      notifyUser(data.message,"success");
-           
+      notifyUser(data.message, "success");
+      toast.success("Registration Successful, please check your email to activate your account");
+      navigate('/login');
     },
-    onError: (error:any) => {
-      const data:any = error.response.data
-      notifyUser(data.message.error,"error");
+    onError: (error: any) => {
+      const data: any = error.response.data;
```

Comment:
How are we sure user would understand the backend error message incase a more robust error message was received?

Let craft more elegant error and user friendly error mesage instead.

Reply:
@XKolz